docs/pages/access-controls/sso/google-workspace.mdx

```diff
@@ -170,10 +170,25 @@ correct Google Workspace plan.
 
 ### Create an OAuth client ID
 
-In the [Create an OAuth client ID](https://console.cloud.google.com/apis/credentials/oauthclient) page of the GCP console, select "Web application" as the Application type, pick a name, then add the following as an authorized redirect URI.
+#### Determine your callback URL
+
+You'll need a callback URL in order to complete the creation of an OAuth client ID with Google. For most deployments, this URL is your ```public_addr``` value. This consists of a domain name and port number. 
+
+You can lookup the ```public_addr``` of your Teleport proxy using the following command:
+
+```
+curl https://example.teleport.com/webapi/ping | jq '.proxy'
```

Comment:
Bit of a chicken and egg thing.  This `curl` doesn't include a port like `3080` or `443`.  We should probably include the `3080` example.  I would also make this an include that can be reused.

Reply:
Will open a new PR for this